mixer/pkg/runtime/handler/logger.go

```diff
@@ -84,3 +65,28 @@ func (l logger) Errorf(format string, args ...interface{}) error {
 	l.s.Error(s)
 	return errors.New(s)
 }
+
+// Debugf from adapter.Logger.
+func (l logger) Debugf(format string, args ...interface{}) {
+	l.s.Debugf(format, args...)
+}
+
+// InfoEnabled from adapter.Logger.
+func (l logger) InfoEnabled() bool {
+	return l.l.Core().Enabled(zap.InfoLevel)
+}
+
+// WarnEnabled from adapter.Logger.
+func (l logger) WarnEnabled() bool {
+	return l.l.Core().Enabled(zap.WarnLevel)
+}
+
+// ErrorEnabled from adapter.Logger.
+func (l logger) ErrorEnabled() bool {
```

Comment:
Do we need this ? Errors should always be enabled right ?

Reply:
Well, it's symmetrical so that all levels have this and it matches pkg/log's interface. The user can in fact opt to turn error logging off if they so chose, although I do admit that would be unlikely.